opencti-platform/opencti-graphql/src/domain/xtm-hub.ts

```diff
@@ -65,3 +66,51 @@ export const checkXTMHubConnectivity = async (context: AuthContext, user: AuthUs
 
   return { status: newRegistrationStatus };
 };
+
+export const autoRegisterOpenCTI = async (context: AuthContext, input: AutoRegisterInput): Promise<string> => {
+  const xtmHubUrl = nconf.get('xtm:xtmhub_url');
+
+  const response = await fetch(`${xtmHubUrl}/graphql-api`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'XTM-Hub-Platform-Token': input.platform_token ?? '',
+      'XTM-Hub-Platform-Id': input.platform_id ?? ''
```

Comment:
Why do we need to have platformId in input ? The platform Id is known on backend side.